eve/get_product_version.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+SCRIPT_FULL_PATH=$(readlink -f "$0")
+REPOSITORY_DIR=$(dirname "$SCRIPT_FULL_PATH")/..
+source "$REPOSITORY_DIR/VERSION" && \
+    echo "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}"
```

Comment:
Does this mean artifacts can not promote RC's or alphas?

Reply:
it can, it's just that artifacts only care about digits, then you can add whatever you want after them.